merlin/dag/executors.py

```diff
@@ -300,6 +311,10 @@ def __init__(self, client=None):
         if client is not None:
             set_client_deprecated(client, "DaskExecutor")
 
+    @property
+    def target_format(self):
+        return DataFormats.PANDAS_DATAFRAME | DataFormats.CUDF_DATAFRAME
+
```

Comment:
I think this is a stray change that's not required yet (until a subsequent PR re: running Systems ops in the `DaskExecutor`.)